### PR TITLE
Add Emailforms

### PR DESCRIPTION
**Pull request by Devilla**

### CODE REVIEW

File: src/sagas/auth.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ export function* fetchRoles() {
 export function* affiliateSuccess() {
   try {
     yield put(load());
-    const res = yield call(api.GET, 'user/sendmail/affiliate');
+    const res = yield call(api.GETAUTH, 'user/sendmail/affiliate?name=shanky&email=sh@ok.com');
```
Comment (ShankyRana) [inline src/sagas/auth.js]: this has to be dynamic

File: src/sagas/auth.js [hunk 2]
```diff
@@ -129,7 +129,7 @@ export function* affiliateSuccess() {
 export function* demoSuccess() {
   try {
     yield put(load());
-    const res = yield call(api.GET, 'user/sendmail/demo');
+    const res = yield call(api.GETAUTH, 'user/sendmail/demo?firstname=shanky&lastname=rana&email=sh@ok.com');
```
Comment (ShankyRana) [inline src/sagas/auth.js]: dynamic

**Comment by ShankyRana** [comment]
PR with development

**Comment by Devilla** [comment]
Done.

**Comment by Devilla** [comment]
New PR.

**Comment by Devilla** [comment]
PR for merge.

**Pull request merged**